src/System.Drawing.Common/tests/IconTests.cs

```diff
@@ -425,9 +435,17 @@ private void ExtractAssociatedIcon_FilePath_Success(string filePath)
 
         [ActiveIssue(20884, TestPlatforms.AnyUnix)]
```

Comment:
Can we do something similar in Icon.Unix.cs when the file doesn't exist?

Reply:
Probably. The Icon class can be further unified I think. As we fix the rest of the P/Invokes it will be much easier to unify the implementations.

Reply:
Cool. We can follow up after this PR on that then.